pctiler/pctiler/colormaps/modis.py

```diff
@@ -0,0 +1,116 @@
+from typing import Dict
+
+from rio_tiler.types import ColorMapType
+
+lulc_colormaps: Dict[str, ColorMapType] = {
+    "modis-10A1": [
+        [[0,25],[43, 140, 190, 255]],
+        [[25,50],[116, 169, 207, 255]],
+        [[50,75],[189, 201, 225, 255]],
+        [[75,100],[241, 238, 246, 255]],
+        [[101,255],[0, 0, 0, 0]], 
+    ],
+    "modis-10A2": {
+        0: (0, 0, 0, 0),
+        1: (0, 0, 0, 0),
+        11: (99, 99, 99, 255),
+        25: (20, 91, 22, 255),
+        37: (0, 11, 128, 255),
+        39: (0, 11, 128,255),
+        50: (128, 128, 128, 255),
+        100: (0, 254, 237, 255),
+        200: (204, 115, 255, 255),
+        254: (0, 0, 0, 0),
+        255: (0, 0, 0, 0),
+    },
+    "modis-13A1/Q1": [
```

Comment:
```suggestion
    "modis-13A1|Q1": [
```
Since these are used in urls, using a `|` instead of `/` feels safer (might not matter, but doesn't hurt).